backend-engineer/importer/README.MD

```diff
@@ -0,0 +1,109 @@
+**Importer Challenge - Backend Engineering @ Vizzuality**
+
+---
+
+Instructions for completing and submitting the challenge:
+First things first. This coding challenge is about creating a space for you to share with us how you work and reason, what things you care about when doing coding work, and how you approach problem-solving.
+
+As such, through this challenge we are not expecting to check if you know the finest algorithms, have all the right answers to a given situation or if you are the best coder in the world. We believe there are no right or wrong answers, so please make yourself comfortable and focus on what you know best.
+
+## ✨ Overview:
+
+Design and implement a robust data ingestion and querying system using a modular backend architecture. 
+The goal is to simulate a real-world import pipeline that scales well and meets performance, quality, and maintainability standards.
+
+Please read carefully all the instructions. If you feel that something is not clear, please reach out, or feel free to make an educated guess!
+
+
+## 📆 Time estimate
+
+Please submit your ideas within 7 days of receiving the challenge, so that we have enough time to review your solution with the rest of the team
+before the next interview. 
+
+Based on our previous experience, and the available tooling nowadays, we believe that it should take no more than 8-10h.
+But do not let this to be a limiting factor to you. Good enough means good enough, but it's up to you how much do you want to polish it
+
+---
+
+## 1. Requirements
+
+You are required to implement a small system that allows importing and querying environmental data.
+
+This system must include:
+
+### A. **CSV Import Microservice**
+
+* Accepts a CSV upload (e.g., via POST endpoint).
+* Parses and validates the file.
+* Stores the data in a **relational database**.
+* Performs basic aggregation (e.g., record count, min/max for numeric fields) post-import.
+
+### B. **Query API Microservice**
+
+* Exposes a REST API to query the imported dataset.
+* Must support:
+    * Filtering by any field (query params)
+    * Pagination and sorting
+* Must return results quickly.
+
+### C. **Infrastructure & Tooling**
+
+* Use a single **public** GitHub repository (no need to create separate repositories for the services)
+* Provide a comprehensive commit tree
+* Use **Docker + docker-compose** to deliver a working setup. It 
```

Comment:
I suggest using the latest text from the importer-large-files challenge:

Provide a way to run your application with Compose and document how to do so.

Reply:
Yeh, I had no problem with the previous text, but I only wanted to make this very clear because it's been a common thing for candidates to deliver non-functional docker setup (for any arbitrary reason) which then takes extra time for the reviewer to debug, tests etc... Also I think if it's a requirement, it must be delivered properly, so I wanted to be very specific with this, but I don't mind rolling it back

Let's see what the rest of the team thinks?

Reply:
I think it's fine the way you have it. It's specific but not overly verbose, so totally fine.